RULES.md

```diff
@@ -2181,17 +2182,53 @@ A route's color and `route_text_color` should be contrasting.
 
 </details>
 
-<a name="RouteShortAndLongNameEqualNotice"/>
+<a name="RouteLongNameConainsShortName"/>
 
-### route_short_and_long_name_equal
+### route_long_name_contains_short_name
 
-A single route has the same values for `route_short_name` and `route_long_name`.
+Long name can not contain short name for a single route.
 
 Example of bad data:
 
 | `route_id` 	| `route_short_name` 	| `route_long_name` 	|
 |------------	|--------------------	|-------------------	|
 | route1     	| L1                 	| L1                	|
+| route2     	| L1                 	| L1 Long Name      	|
+| route3     	| L1                 	| L1- Long Name      	|
+| route3     	| L1                 	| L1 - Long Name     	|
+| route4     	| L1                 	| L1 (Long Name)     	|
+| route5     	| L1                 	| L1) Long Name     	|
```

Comment:
Let me know if these examples suffice, and/or are representative of bad and good data...

Reply:
Please see my suggestion above. I think the users will get the point with the 3-row table above, although the validator should flag these examples as bad data.